challenges/objects-arrays.js

```diff
@@ -59,12 +99,18 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+for (let i = 0; i < graduates.length; i++) {
+  contactInfo.push(graduates[i].first_name, graduates[i].email)
+}
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
-const uni = [];
-console.log(uni);
+// const uni = [];
+// for (let i = 0; i < graduates.length; i++) {
+// if (graduates[i].university.includes('Uni'),
+//   uni.push(graduates[i])};
+// console.log(uni);
```

Comment:
this one is really close but just has some parentheses/curly brace syntax issues going on